src/pages/ImageMapProView/ImageMapProView.jsx

```diff
@@ -21,32 +21,68 @@ import {
 } from '../../utils/waitForElementToBeAppended';
 
 import './ImageMapPro.scss';
+import HoursModal from '../../components/HoursModal/HoursModal';
 
 const ImageMapProView = () => {
```

Comment:
when clicking a reserved desk and closing the modal after that, it changes the color of the desk to green